config/routes.rb

```diff
@@ -33,7 +33,7 @@
 
   resources :documents, controller: :documents do
     collection do
-      DocumentNavigation.all_controllers.uniq.each do |controller_class|
+      (DocumentNavigation.all_controllers + [Documents::SendRequestedDocumentsController]).uniq.each do |controller_class|
```

Comment:
This logic (appending the requested docs controller) should be handled by the Navigation class, not the routes file. `.all_controllers` is intended to include any controllers, even if they are not in the main flow.